corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/util.js

```diff
@@ -75,6 +75,27 @@ hqDefine("cloudcare/js/formplayer/menus/util", function () {
         FormplayerFrontend.regions.getRegion('breadcrumb').show(breadcrumbView);
     };
 
+    var showLanguageMenu = function (langs) {
+        var localeModels,
+            localeCollection;
+
+        FormplayerFrontend.regions.addRegions({
+            formMenu: "#form-menu",
+        });
+        localeModels = _.map(langs, function (lang) {
```

Comment:
Is there a reason most identifiers are named with "language" but a few use "locale"?

Reply:
Formplayer code that dealt with this mostly called it a locale (although it does refer to langs in some places) and existing web apps code was inconsistent. I started using locale in most places, but some langs slipped through. In particular, in this case, I was using the `locales` (which is what we call them at the app level) property of the menuResponse, before I realized that when it corresponds to a form there is a separate property called `langs` that was a little nicer (did not include a "default"). The localeModels is probably a holdover from when I was iterating over locales rather than langs. Happy to make this slightly more consistent, although not certain which one is better.

Reply:
Consistency within formplayer would be good, consistency across HQ would be great. My bias is to favor langs because that's what app manager uses, and that's unlikely to change (since couch attributes are a pain to rename). But I leave it to your judgment how much, if anything, to change in this PR.